string  node_name
float32 cpu_load
